theano/scalar/basic.py

```diff
@@ -798,10 +797,10 @@ def __mul__(self, other):
         return mul(self, other)
 
     def __truediv__(self, other):
-        return div_proxy(self, other)
+        return true_div(self, other)
 
-    def __div__(self, other):
-        return div_proxy(self, other)
+    def __intdiv__(self, other):
+        return int_div(self, other)
```

Comment:
Can one of the old tests be used to cover this?

Reply:
What exactly is `__intdiv__`?  Shouldn't this actually be `__div__`?  If not, where's the `__div__` implementation?